### DISCUSSION TITLE
Small contribution

### DISCUSSION THREAD

**Discussion by AgroMeester** [Mogli12/AIVehicleExtension#270] [Ideas]
Hi Stefan,

Saw you were working on this excellent mod so here's another small contribution.
Use if you like!

Greetings,

AgroMeester

![aive-v1](https://user-images.githubusercontent.com/25507803/55117601-b07c8d00-50eb-11e9-9093-03c02031b0c4.png)

**Comment by Mogli12** [comment]
Very nice icon! Many thanks!
